src/Firehose.Web/Web.config

```diff
@@ -37,10 +36,6 @@
   <system.web>
     <compilation debug="true" targetFramework="4.5.1">
       <assemblies>
-        <add assembly="System.Globalization, Version=4.0.0.0,
```

Comment:
I have not touch this. I have only add my AlessandroCaliaro.cs.
What Should I do?